docs/source/projections/cass.rst

```diff
@@ -1,10 +1,142 @@
 
+Although the Cassini projection has been largely replaced by the Transverse Mercator, it is still in limited use outside the United States and was one of the major topographic mapping projections until the early 20th century.
+
++---------------------+-------------------------------------------------------------------------+
+| **Classification**  | Transverse and oblique cylindrical                                      |
++---------------------+-------------------------------------------------------------------------+
+| **Available forms** | Forward and inverse, Spherical and Elliptical                           |
++---------------------+-------------------------------------------------------------------------+
+| **Defined area**    | Global, but best used near the central meridian with long, narrow areas |
++---------------------+-------------------------------------------------------------------------+
+| **Implemented by**  | Gerald I. Evenden                                                       |
++---------------------+-------------------------------------------------------------------------+
+| **Options**                                                                                   |
++---------------------+-------------------------------------------------------------------------+
+| `+lat_0`            | Center of the map : latitude of origin                                  |
++---------------------+-------------------------------------------------------------------------+
+
 .. image:: ./images/cass.png
    :scale: 50%
    :alt:   Cassini  
 
+Usage
+########
+
+There has been little usage of the spherical version of the Cassini, but the ellipsoidal Cassini-Soldner version was adopted by the Ordnance Survey for the official survey of Great Britain during the second half of the 19th century (Steers, 1970, p. 229). Many of these maps were prepared at a  scale of 1:2,500. The Cassini-Soldner was also used for the detailed mapping of many German states during the same period. 
+
+
+Example using EPSG 30200 (Trinidad 1903, units in clarke's links)::
+
+    $ echo 0.17453293 -1.08210414 | proj +proj=cass +lat_0=10.44166666666667 +lon_0=-61.33333333333334 +x_0=86501.46392051999 +y_0=65379.0134283 +a=6378293.645208759 +b=6356617.987679838 +to_meter=0.201166195164 +no_defs 
+    66644.94	82536.22
+
+Example using EPSG 3068 (Soldner Berlin)::
+
+    $ echo 13.5 52.4 | proj +proj=cass +lat_0=52.41864827777778 +lon_0=13.62720366666667 +x_0=40000 +y_0=10000 +ellps=bessel +datum=potsdam +units=m +no_defs
+    31343.05	7932.76
+
+
+Mathematical definition
+#######################
+
+The formulas describing the Cassini projection are all taken from proj4 sources.
+
+:math:`\phi_0` is the latitude of origin that match the center of the map. It can be set with ``+lat_0``.
+
+
+Spherical form
+==============
+
+Forward projection
+------------------
+
+.. math::
+
+  x = \arcsin(\cos(\phi) * \sin(\lambda))
```

Comment:
The * here looks kinda un-mathematical when rendered. I think it would look better if it is left out, or maybe instead use something like \cdot or \times depending on how the end result turns out.